### PR TITLE
MassInstallAction: Install the GitHubActions workflow on this repository

### PR DESCRIPTION
**Pull request by fchorney**
This pull request sets up the GitHubActions workflow on this repository. cc: @fchorney

### CODE REVIEW

**Comment by christopher-dG** [comment]
@fchorney this one can be closed, right?

**Comment by fchorney** [comment]
@christopher-dG do we not want to update the GHA CI scripts?

**Comment by christopher-dG** [comment]
I was mostly thinking that this repo didn't need the Slack and nightly stuff since I'm mostly the one who does maintenance, but if you guys want it that's no problem (especially if it helps you guys maintain it too!).

**Comment by fchorney** [comment]
> I was mostly thinking that this repo didn't need the Slack and nightly stuff since I'm mostly the one who does maintenance, but if you guys want it that's no problem (especially if it helps you guys maintain it too!).

ah good point. I'll see what people think.

File: test/runtests.jl [hunk 1]
```diff
@@ -61,8 +61,9 @@ mktempdir() do dir
 
                 # Quite a bit of output depends on the Julia version,
                 # and the test fixtures are made with Julia 1.5.
+                # These tests fail on 32-bit
```
Comment (mattBrzezinski) [inline test/runtests.jl:64]: What's the reason for these failing on 32-bit? Do we have an issue logged for it? Or is it even worth logging / already know elsewhere?

Comment (christopher-dG) [reply test/runtests.jl:64]: I wasn't aware of these failures, and I can't really think of why they would break there. So it's a good test to have!

Comment (fchorney) [reply test/runtests.jl:64]: I am currently unsure why they fail on 32-bit. They seem to be failing in a spot that's meant to fail but works fine on 64-bit? I didn't look into it super deep

Comment (fchorney) [reply test/runtests.jl:64]: for reference (hah) it was failing here only on 32-bit for whatever reason https://github.com/invenia/PkgTemplates.jl/blob/master/test/reference.jl#L50

Comment (christopher-dG) [reply test/runtests.jl:64]: If you can find a link to the job logs, that would help a lot (or just enable those tests again for 32-bit)

Comment (fchorney) [reply test/runtests.jl:64]: Should be in this run https://github.com/invenia/PkgTemplates.jl/pull/253/checks?check_run_id=1526431542

Comment (fchorney) [reply test/runtests.jl]: Thanks for the fix!

**Review by mjram0s — APPROVED** [review]

**Comment by mattBrzezinski** [comment]
:+1:

**Pull request merged**